src/utils/rest-api.ts

```diff
@@ -537,6 +540,27 @@ export function createContingencyList(
         body: JSON.stringify(formContent),
     });
 }
+export function createFilterBasedContingency(
+    name: string,
+    description: string,
+    filtersUuids: FilterAttributes[],
+    parentDirectoryUuid: UUID | undefined
+) {
+    console.info('Creating a new filter based contingency list...');
+    const urlSearchParams = new URLSearchParams();
+    urlSearchParams.append('description', description);
+    urlSearchParams.append('parentDirectoryUuid', parentDirectoryUuid ?? '');
+
+    const createContingencyListUrl = `${PREFIX_EXPLORE_SERVER_QUERIES}/v1/explore${CONTINGENCY_ENDPOINTS.FILTERS_CONTINGENCY_LISTS}/${encodeURIComponent(
+        name
+    )}?${urlSearchParams.toString()}`;
+    console.debug(createContingencyListUrl);
+
+    return backendFetch(createContingencyListUrl, {
+        method: 'post',
+        body: JSON.stringify({ filters: filtersUuids }),
```

Comment:
```suggestion
        body: JSON.stringify(contingency),
```
Should avoid build object in api method because we need typing correctly at params of api method